packages/tasks/src/model-libraries.ts

```diff
@@ -616,6 +616,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"Open_Sora_v2.safetensors"`,
 	},
+	outetts: {
```

Comment:
requires a few models under https://huggingface.co/models?other=outetts before getting this PR merged